std::string vehicle;
std::string bag_time;
int sensor_num;
int frame_num;
std::vector<std::string> sensor_vec_;
pointcloud_tools::SensorDataID init_id;
std::string root_dirname_ = "/home/rideflux/v5_1_sample_data_1/";
std::string config_filename_ = root_dirname_+"configuration.txt";
int max_iteration;
int edge_distance_threshold;
bool is_single_lidar_imu_graph = false;
bool is_multi_lidar_imu_graph1 = false;
bool is_multi_lidar_imu_graph2 = false;
